src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -263,6 +264,11 @@
 					.examples("on tool break:")
 					.since("2.1.1");
 		}
+		Skript.registerEvent("Item Damage", SimpleEvent.class, PlayerItemDamageEvent.class, "item damag(e|ing)")
+				.description("Called when a player damages their tool by using it, ie: mining, chopping wood or using shears on a sheep.")
```

Comment:
Description should be updated to mention armor as well. :)

Reply:
Right yes, totally missed that.